src/main/java/frc/robot/Controls.java

```diff
@@ -569,14 +585,19 @@ private void configureClimbPivotBindings() {
       s.climbPivotSubsystem.isClimbing().whileTrue(setClimbLEDs);
     }
 
+    // regularly run the advanced climb check
     s.climbPivotSubsystem.setDefaultCommand(
         s.climbPivotSubsystem.advanceClimbCheck().withName("Advance Climb Check"));
 
+    // check if the climb controller is connected, and whne start is pressed move to the next climb
+    // position
     connected(climbTestController)
         .and(climbTestController.start())
         .onTrue(s.climbPivotSubsystem.advanceClimbTarget());
+    // on start or right trigger, move to climb out or climbed respectively on operator
     operatorController.start().onTrue(s.climbPivotSubsystem.toClimbOut());
     operatorController.rightTrigger().onTrue(s.climbPivotSubsystem.toClimbed());
+    // manual control for climb test controller for - direction
```

Comment:
"minus direction" or "negative direction" would be clearer than "- direction". It'd also be nice to add a comment about which physical direction it is (out/in or up/down).